src/utils/img_helpers.py

```diff
@@ -74,6 +74,11 @@ def initialize_globals(path_list: list[Path]) -> None:
     global_vars.X_CENTER = get_middle_dimension(curr_img, View.X)
     global_vars.Y_CENTER = get_middle_dimension(curr_img, View.Y)
 
+    global_vars.BINARY_THRESHOLD_FILTER.SetLowerThreshold(100)
+    global_vars.BINARY_THRESHOLD_FILTER.SetUpperThreshold(200)
+    global_vars.BINARY_THRESHOLD_FILTER.SetInsideValue(0)
+    global_vars.BINARY_THRESHOLD_FILTER.SetOutsideValue(1)
```

Comment:
On Hank's laptop, these two values work. On my laptop, I need to flip them like so for contouring (i.e., click Apply) to work. Very weird issue...

```py
    global_vars.BINARY_THRESHOLD_FILTER.SetInsideValue(1)
    global_vars.BINARY_THRESHOLD_FILTER.SetOutsideValue(0)
```

Reply:
We were using different threshold settings. Fixed by 8eb5383, though the background color detection function could be improved.